### ISSUE TITLE
Disable LDAP Users not having a role anymore

### ISSUE THREAD

**Issue by sc911** [sqlpad/sqlpad#1013]
When using role-based LDAP-Auth I would expect that when a user who previously matched any of the both role filters gets disabled if there is no match at some point of time in the future.
With `SQLPAD_LDAP_AUTO_SIGN_UP=true` LDAP-auth creates a new user in the user database if LDAP login is successful and assigns the role of the matching filter.
If the role changes (admin->editor or editor->admin) the role of the local user gets updated.
But if non of the role-filters matches there is no update on the local user!
From my point of view the user should be disabled in that case.
So basically there is currently no way to do user administration purely in LDAP.
Is there any chance on an update to this? If I find some time I will check the code
